src/server/views/prototypes/mot-test/index.njk

```diff
@@ -0,0 +1,202 @@
+
+      </header>
+
+      <header class='content-header'>
+
+        <ul class='content-header__tertiary-list u-margin-bottom-35'>
+
+          <li class='content-header__list-item--primary'>BHW, 790</li>
+
+          <li class='content-header__list-item'>AJ30CTK</li>
+
+          <li class='content-header__list-item'>First used 25 Apr 2005</li>
+
+        </ul>
+
+      </header>
+      
+      <div class='text u-clearfix'>
+        
+        <p><a href='/prototypes/'>Print the MOT inspection sheet</a> for this test.</p>
+
+        <p>See <a href='/prototypes/'>testing advice</a> for this vechicle.</p>
+
+      </div>
+
+      <div class="action-panel">
+
+        <div class="clear-float">
+
+          <div class="action-panel__container">
+
+            <h3 class="action-panel__heading">Odometer</h3>
+
+          </div>
+
+          <div class="action-panel__container">
+
+            <h3 class="action-panel__value">Not recorded</h3>
+
+          </div>
+
+          <div class="action-panel__container--last">
+
+            {{ components.button(text='Add reading', url='/prototypes/annual-assessment-admin-tool/send-reminders/review') }}
+
+          </div>
+
+        </div>
+
+      </div>
+
+      <div class="action-panel">
+
+        <div class="clear-float">
+
+          <div class="action-panel__container">
+
+            <h3 class="action-panel__heading">Defects</h3>
+
+          </div>
+
+          <div class="action-panel__container--last">
+
+            {{ components.button(text='Add a defect', url='/prototypes/mot-test/categories/') }}
+
+          </div>
+
+        </div>
+
+      </div>
+
+      <div class="action-panel">
+
+        <div class="clear-float">
+
+          <div class="action-panel__container">
+
+            <h3 class="action-panel__heading">Break test</h3>
+
+          </div>
+
+          <div class="action-panel__container">
+
+            <h3 class="action-panel__value">Not recorded</h3>
+
+          </div>
+
+          <div class="action-panel__container--last">
+
+            {{ components.button(text='Add break tests', url='/prototypes/annual-assessment-admin-tool/send-reminders/review') }}
+
+          </div>
+
+        </div>
+
+      </div>
+
+      <div class="action-panel">
+
+        <div class="clear-float">
+
+          <div class="action-panel__container--twothirds">
+
+            <h3 class="action-panel__heading">Testers comments</h3>
+
+          </div>
+
+          <div class="action-panel__container--last">
+
+            {% if viewData.comment %}
+
+              {{ components.button(text='Edit comment', url='/prototypes/mot-test/edit-tester-comment/edit/1') }}
+
+            {% else %}
+
+              {{ components.button(text='Add comment', url='/prototypes/mot-test/add-tester-comment/') }}
+
+            {% endif %}
+
+          </div>
+
+        </div>
+
+      </div>
+
+      {% if viewData.comment %}
+
+        <div class="defect-summary">
```

Comment:
Why are you using a defect-summary list here.

Please create a new element which is just tester comments.

You only need a wrapper, paragraph and edit buttons.